### ISSUE TITLE
You dont have permission

### ISSUE THREAD

**Issue by smart-reports** [staart/ui#754]
Hi Anand, here is the behaviour we're seeing now

User registers and verifies email - works fine now
User tries to login - immediately presented with a "You dont have permission to perform this action"....
Press F5 to refresh and it looks like the user is logged in - 
However, when I start to click on the user icon the top right and select team or user settings - it keeps presenting us with this "You dont have permission".  

Must be something we are not doing?

![image](https://user-images.githubusercontent.com/50381700/77312018-1862b500-6cf9-11ea-836d-d46c07837775.png)

**Comment by AnandChowdhary** [comment]
This might be a weird solution, but can you (i) restart your server and (ii) log out and log in? If that fixes it, I think I know what's going wrong.

**Comment by ginocbjr** [comment]
Just to add it is also happening in your test page. I just registered this account. 
https://staart-demo.o15y.com/auth/login
ogremagi07@gmail.com
Password123

**Comment by AnandChowdhary** [comment]
Yep, I just tested it and got the same result. I'm restarting the server now (by running `npm run start`) to see if that fixes it. If it does, I know what's going wrong.

**Comment by AnandChowdhary** [comment]
That didn't do it. This is a super high priority bug.

**Comment by AnandChowdhary** [comment]
It's failing in authorization here: https://github.com/staart/api/blob/master/src/rest/membership.ts#L25. I'll move this issue to staart/api instead.
